### ISSUE TITLE
Password Reset Error

### ISSUE THREAD

**Issue by sanfordd** [chemheritage/chf-sufia#157]
Attempting to use the password reset template once the e-mail has been configured generates an error. Attached below is the extracted sufia log for the attempt. And here's the key line
F, [2016-02-10T12:46:52.581959 #4027] FATAL -- : 
ActionView::Template::Error (Missing host to link to! Please provide the :host parameter, set default_url_options[:host], or set :only_path to true):
[password-reset-error-log.txt](https://github.com/chemheritage/chf-sufia/files/125477/password-reset-error-log.txt)

**Comment by sanfordd** [comment]
TLS is working, but we're now onto a brand new issue Here's the excerpted log sample:

D, [2016-02-16T14:16:15.135412 #32018] DEBUG -- : (0.3ms)  SELECT DISTINCT COUNT(DISTINCT "mailboxer_conversations"."id") FROM "mailboxer_conver sations" INNER JOIN "mailboxer_notifications" ON "mailboxer_notifications"."conversation_id" = "mailboxer_conversations"."id" AND "mailboxer_notifications"."type" IN ('Mailboxer::Message') INNER JOIN "mailboxer_receipts" ON "mailboxer_receipts"."notification_id" = "mailboxer_notifications"."id" WHERE "mailboxer_notifications"."type" = 'Mailboxer::Message' AND "mailboxer_receipts"."mailbox_type" = $1 AND "mailboxer_receipts"."trashed" = 'f' AND "mailboxer_receipts"."deleted" = 'f' AND "mailboxer_notifications"."type" = 'Mailboxer::Message' AND "mailboxer_receipts"."receiver_id" = $2 AND "mailboxer_receipts"."receiver_type" = $3 AND "
mailboxer_receipts"."is_read" = 'f'  [["mailbox_type", "inbox"], ["receiver_id", 3], ["receiver_type", "User"]]
D, [2016-02-16T14:16:15.136838 #32018] DEBUG -- :   Mailboxer::Conversation Load (0.3ms)  SELECT DISTINCT "mailboxer_conversations".* FROM "mailboxer_conversations" INNER JOIN "mailboxer_notifications" ON "mailboxer_notifications"."conversation_id" = "mailboxer_conversations"."id" AND "mailboxer_notifications
/FATAL
...skipping
D, [2016-02-16T16:15:54.669586 #32018] DEBUG -- : Date: Tue, 16 Feb 2016 16:15:54 -0500I, [2016-02-16T16:15:54.669807 #32018]  INFO -- : Completed 500 Internal Server  Error in 17ms (ActiveRecord: 1.8ms)
F, [2016-02-16T16:15:54.672281 #32018] FATAL -- :

**Comment by hackmastera** [comment]
So the error i'm currently getting is the same as the original error. Config change is still in place and I'm not really sure how to debug this. Will keep at it.

**Comment by hackmastera** [comment]
Looks like I was in the config directory for an old deployment; this change was never made in the master branch (was just local to server) and so is not yet permanent.

**Comment by hackmastera** [comment]
Okay now I am getting the following:

I, [2016-02-17T16:10:21.436335 #25586]  INFO -- : Completed 500 Internal Server Error in 360ms (ActiveRecord: 2.8ms)
F, [2016-02-17T16:10:21.438643 #25586] FATAL -- : 
Net::SMTPServerBusy (454 4.7.0 TLS not available due to local problem
):
  /usr/local/lib/ruby/2.2.0/net/smtp.rb:964:in `check_response'
  /usr/local/lib/ruby/2.2.0/net/smtp.rb:933:in `getok'

**Comment by hackmastera** [comment]
this is resolved at the application level; waiting on an accompanying change to deploy scripts.
